tests/alerts/alert_test_suite.py

```diff
@@ -173,7 +173,6 @@ def verify_saved_events(self, found_alert, test_case):
         if self.deadman:
             return
 
-        assert len(found_alert['_source']['events']) == len(test_case.full_events)
```

Comment:
I'm removing this since this check doesn't work if the sample event number != threshold, now that I think about it, it might not be providing much value in the first place.